### PR TITLE
Feature/xinnanli/community typo fix

### PR DESCRIPTION
**Pull request by xinnanlisf**
Merging pull request from community #483

### CODE REVIEW

**Comment by xinnanlisf** [comment]
@lianeyee 

Could you release this release note? Thanks!

**Comment by lianeyee** [comment]
@xinnanlisf I made a slight edit. It's best if we can describe the issue specifically enough that the person who filed the issue (especially external community folks) can tell that it was their bug that we fixed. So I added a link to the affected page and mentioned where the typo was. Just helps to further encourage and recognize community contributions. Thanks!

**Comment by dylanhansen** [comment]
@xinnanlisf Let's also add a quick thank you in that line to thank Pedro Dal Col for his contribution.  Thanks!

File: ApexDocContent/TDTM.htm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <html>
     <body>
         <h2>Table-Driven Trigger Management (TDTM) Classes</h2>
-        <p>These are the classes that support our table-driven trigger management approach, where only on trigger exists 
+        <p>These are the classes that support our table-driven trigger management approach, where only one trigger exists
```
Comment (xinnanlisf) [inline ApexDocContent/TDTM.htm]: @pdalcol

Thank you for your contributions, great job!

**Pull request merged**